lib/DB/Schema/ResultSet/Course.pm

```diff
@@ -323,7 +319,6 @@ sub getGlobalSetting ($self, %args) {
 		unless $global_setting;
 	return $global_setting if $args{as_result_set};
 	my $setting_to_return = { $global_setting->get_inflated_columns };
-	$setting_to_return->{default_value} = $setting_to_return->{default_value}->{value};
```

Comment:
we can ditch the line above this one as well and just `return { $global_setting->get_inflated_columns };`

Reply:
fixed.